### PR TITLE
Card size smaller with no commit information (#2476)

### PR DESCRIPTION
**Pull request by keithchong**
See 2476 for more details

Signed-off-by: Keith Chong <kykchong@redhat.com>

### CODE REVIEW

**Review by ciiay — CHANGES_REQUESTED** [review]
Seems like it's missing locales for the new contents. Other than that looks good to me. Left two small comments.

File: src/components/EnvironmentDetailsPage.tsx [hunk 1]
```diff
@@ -42,7 +42,10 @@ const EnvironmentDetailsPage: React.FC<EnvironmentDetailsPageProps> = ({ match,
         } catch (err) {
           setError(err);
         }
-        setEnvsData(data);
+        console.log("data is " + data);
```
[the remark this answers is not in the document]
Comment (keithchong) [reply src/components/EnvironmentDetailsPage.tsx]: I removed it.

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "http-server": "./http-server.sh ./dist",
     "clean": "rm -rf dist",
     "build": "yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack",
+    "dev": "yarn clean && REACT_REFRESH=true NODE_OPTIONS=--max-old-space-size=4096 yarn ts-node ./node_modules/.bin/webpack serve --mode=development --progress",
```
[the remark this answers is not in the document]
Comment (keithchong) [reply package.json]: This was the dev mode just like what we had in the static plugin. I was using it and it seemed ok to me.  It's like `start`.   I've removed it.

**Comment by ciiay** [comment]
LGTM.
We still need to update local files for other languages too. Shall we keep a note somewhere?

**Review by ciiay — APPROVED** [review]

**Pull request merged**